.bazelrc

```diff
@@ -12,8 +12,7 @@ test --test_env=LANG=C.UTF-8 --test_env=LOCALE_ARCHIVE
 
 # Platform / Toolchain Selection
 # ------------------------------
-build:linux-nixpkgs --host_platform=@io_tweag_rules_nixpkgs//nixpkgs/platforms:host
-build:macos-nixpkgs --host_platform=@io_tweag_rules_nixpkgs//nixpkgs/platforms:host
+build:nixpkgs --host_platform=@io_tweag_rules_nixpkgs//nixpkgs/platforms:host
```

Comment:
This breaks the config flags documented in the README. With this change the last reference to `linux-nixpkgs` is gone and `--config=linux-nixpkgs` becomes invalid. I'd prefer to keep the config matrix in the README valid for symmetry with the bindist cases and in case we need platform specific flags in the future.

Reply:
Bazel doesn't appear to support empty configurations (would make a good feature request upstream). We could hack around that, but I opted instead to remove this configuration from the README.

Reply:
I've opened such a feature request: https://github.com/bazelbuild/bazel/issues/12844.
I don't think that [change](https://github.com/tweag/rules_haskell/pull/1471/commits/6a7d835b5d2fc7c0baea05780654b79562f4a088) is correct. On Linux/Nixpkgs we would need `--config=nixpkgs` but the README claims `(unneeded)`. On macOS we would now need both `--config=nixpkgs --config=macos-nixpkgs` but the README claims `--config=macos-nixpkgs`. The aim of https://github.com/tweag/rules_haskell/pull/1301 was to require only one top-level `--config` flag for developers.
I've pushed a commit that loads `--config=nixpkgs` from both `linux-nixpkgs` and `macos-nixpkgs`. With this developers only need a single top-level config and both linux-nixpkgs and macos-nixpkgs unfold the nixpkgs config as well.

Reply:
I'm not sure I follow. `--config=nixpkgs` doesn't exist in this branch.

Reply:
Sorry, spurious comment. I had the wrong branch checked out.